dali/test/python/test_operator_slice.py

```diff
@@ -339,37 +350,40 @@ def check_slice_synth_data_vs_numpy(device, batch_size, input_shape, layout, axe
     compare_pipelines(
         SliceSynthDataPipeline(device, batch_size, layout, iter(eiis[0]), iter(eii_args[0]),
             axes=axes, axis_names=axis_names, normalized_anchor=normalized_anchor,
-            normalized_shape=normalized_shape),
+            normalized_shape=normalized_shape, input_type=input_type, output_type=output_type),
         SliceSynthDataPipelinePythonOp(batch_size, layout, iter(eiis[0]), iter(eii_args[1]),
             axes=axes, axis_names=axis_names, normalized_anchor=normalized_anchor,
-            normalized_shape=normalized_shape),
+            normalized_shape=normalized_shape, input_type=input_type, output_type=output_type),
         batch_size=batch_size, N_iterations=5)
 
 def test_slice_synth_data_vs_numpy():
     for device in ["cpu", "gpu"]:
         for batch_size in {1, 8}:
-            for input_shape, layout, axes, axis_names in \
-                [((200,400,3), "HWC", None, "WH"),
-                ((200,400,3), "HWC", None, "HW"),
-                ((200,400,3), "HWC", None, "C"),
-                ((200,400,3), "HWC", (1,0), None),
-                ((200,400,3), "HWC", (0,1), None),
-                ((200,400,3), "HWC", (2,), None),
-                ((200,), "H", (0,), None),
-                ((200,), "H", None, "H"),
-                ((200,400), "HW", (1,), None),
-                ((200,400), "HW", None, "W"),
-                ((80, 30, 20, 3), "DHWC", (2,1,0), None),
-                ((80, 30, 20, 3), "DHWC", (0,1,2), None),
-                ((80, 30, 20, 3), "DHWC", (2,1), None),
-                ((80, 30, 20, 3), "DHWC", None, "WHD"),
-                ((80, 30, 20, 3), "DHWC", None, "DHW"),
-                ((80, 30, 20, 3), "DHWC", None, "WH"),
-                ((80, 30, 20, 3), "DHWC", None, "C")]:
+            for input_shape, layout, axes, axis_names, input_type, output_type in \
```

Comment:
If you want to not explode the number of cases, you can keep the old loop, select one of the cases and add after the loop another one:
```
for input_type in [types.FLOAT, types,UINT32, types.INT32, types.INT64]:
  for output_type in [None, types.FLOAT, types,UINT32, types.INT32, types.INT64]:
    yield check_slice_synth_data_vs_numpy device, batch_size, (200,400,3), "HWC", None, "HW", True, True, input_type, output_type
```
or something similar.